### PR TITLE
create rating

### PR DESCRIPTION
**Pull request by ntdo2506**

### CODE REVIEW

File: src/rating/admin.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+from django.contrib import admin
+from .models import Rating
+
+admin.site.register(Rating)
```
Comment (datnguyen0126) [inline src/rating/admin.py:4]: new line

File: src/rating/models.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
+from django.db import models
+from users.models import CustomUser
+from django.core.validators import MaxValueValidator, MinValueValidator
+import uuid
+
+class Rating(models.Model):
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False, unique=True)
+    user_id = models.ForeignKey(CustomUser, on_delete=models.CASCADE,blank=True)
+    tour_id = models.ForeignKey('tours.Tours', on_delete=models.CASCADE, blank=True)
```
Comment (datnguyen0126) [inline src/rating/models.py:9]: Why don't you import Tours model?

File: src/tours/models.py [hunk 3]
```diff
@@ -22,5 +23,14 @@ class Tours(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
 
+    def avg_rating(self):
+        sum = 0
+        ratings = Rating.objects.filter(tour_id=self)
```
Comment (prd-anh-nguyen) [inline src/tours/models.py]: can u use sum/avg in sql in here?

Comment (ntdo2506) [reply src/tours/models.py]: No, I can't
Because ratings is list so i use for loop to plus sum
And i want to check len(rantings)

Comment (ptran068) [reply src/tours/models.py]: image
Sum is the same

Comment (ntdo2506) [reply src/tours/models.py]: I understand..I just fix it..thank Phong and a.Anh

**Pull request closed without merging**